src/lib.rs

```diff
@@ -28,6 +28,7 @@ pub mod transports;
 pub mod types;
 
 pub use api::{Web3Main as Web3, ErasedWeb3};
+pub use helpers::CallResult;
```

Comment:
Ah, sorry didn't notice that my PR is not merged yet.
This #33 exposes the whole `helpers` module, so docs will be fixed for `CallResult` as well. That's why I think that re-export is not necessary.